ObjectiveGitTests/QuickSpec+GTFixtures.m

```diff
@@ -95,6 +99,24 @@ - (NSString *)pathForFixtureRepositoryNamed:(NSString *)repositoryName {
 }
 
 - (BOOL)unzipFile:(NSString *)member fromArchiveAtPath:(NSString *)zipPath intoDirectory:(NSString *)destinationPath error:(NSError **)error {
+
+#if TARGET_OS_IPHONE
+	// iOS: unzip in-process using SSZipArchive
```

Comment:
That would certainly make things cleaner. My only concern is speed. The iOS tests are quite a bit slower than the Mac tests and I'm not sure why. I'll experiment with using SSZipArchive for the Mac tests to see if they slow down or not.